src/lucky/tags/base_tags.cr

```diff
@@ -30,7 +30,15 @@ module Lucky::BaseTags
       end
     end
 
-    def {{method_name.id}}(boolean_attrs : String = "", options = EMPTY_HTML_ATTRS, **other_options, &block)
+    def {{method_name.id}}(options = EMPTY_HTML_ATTRS, **other_options, &block)
```

Comment:
How do I just make this one call the next override method? 
```crystal
{{method_name.id}}("", options = EMPTY_HTML_ATTRS, **other_options, &block) 
#=> src/lucky/tags/form_helpers.cr:11: can't use `yield` inside a proc literal or captured block
```
Adding a default arg to the second one causes any tag that uses a NamedTuple literal to fail like 
```
div({class: "test"}) #=> no overload matches 'TestRender#header' with type NamedTuple(class: String)